docs/troubleshoot/troubleshoot-zos-certificate.md

```diff
@@ -25,12 +25,57 @@ Dkeystore.pkcs12.keyProtectionAlgorithm=PBEWithSHAAnd3KeyTripleDES -J-
 Dkeystore.pkcs12.keyPbeIterationCount=50000
 ```
 
-- Set the flag `keystore.pkcs12.legacy` enabled with no value to create a PKCS12 keystore that can be loaded.
+- Set the flag `keystore.pkcs12.legacy` to enabled with no value to create a PKCS12 keystore that can be loaded.
 
-**Notes**
+:::note
+* If you already have an existing keystore or you are using keyrings, this error will not happen.
+* If you do not use ZSS, this error will not happen. ZSS is enabled by default.
```

Comment:
This section looks useless, user comes to the troubleshooting section when something happened already, they do not need to know when this will not happen. However we can switch this to a useful information, in case the customer just sees error in the log, like "If you are not planning to use Zowe Desktop then you can disable ZSS component to get rid of this error in the log"